apps/omg/test/omg/conformance/signature_property_test.exs

```diff
@@ -0,0 +1,122 @@
+# Copyright 2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.Conformance.SignaturePropertyTest do
+  @moduledoc """
+  Checks if some properties about the signatures (structural, EIP-712 hashes to be precise) hold for the Elixir and
+  Solidity implementations
+  """
+
+  import Support.Conformance
```

Comment:
please don't import, be explicit in what you're "importing"

Reply:
yeah, didn't do that yet, will do when cleaning up and when I know the layout of modules better.